File: JAVA_inputOutput/inputOutput/src/ForLoop.java
// Dado um numero N, leia N inteiros e imprima a soma deles.
// 1 + 2 + 3 + 4 = 10
import java.util.Scanner;

public class ForLoop {
    public static void main(String[] args) {
        int soma = 0;
        Scanner numberScanner = new Scanner(System.in);
        int number = numberScanner.nextInt();
        numberScanner.close();
        for (int i = 0; i < number + 1; i++){
            soma = soma + i;
        }
        System.out.println(soma);
    }
}

File: CPP_inputOutput/forLoop.cpp
// Dado um numero N, leia N inteiros e imprima a soma deles.
// 1 + 2 + 3 + 4 = 10
#include <iostream>
using namespace std;

int main() {
    int numero = 0;
    int soma = 0;
    cin >> numero;
    for ( int contador = 0; contador < (numero +1); contador++) { //Apanhei aqui, solucao era colocar numero + 1 em paranteses pra resolver a expressão primeiro antes de iterar
        soma = soma + contador;
    }
    cout << soma;
    return 0;
}
